src/configuration-requestor.ts

```diff
@@ -0,0 +1,31 @@
+import { IConfigurationStore } from './configuration-store/configuration-store';
+import { IHttpClient } from './http-client';
+import { BanditParameters, Flag } from './interfaces';
+
+// Requests AND stores flag configurations
+export default class ConfigurationRequestor {
```

Comment:
This file is not new, but renamed and expanded for bandits. However, there are enough changes that GitHub thinks it's new 🤷‍♂️